android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -95,24 +101,48 @@ private static void doLaunch(Context context, HashMap<String, Object> extraData)
 
 
     public static void launch(Context context, HashMap<String, Object> extraData) {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-            PermissionListener listener = new PermissionListener() {
-                @Override
-                public boolean onRequestPermissionsResult(int i, String[] strings, int[] results) {
-                    if (results.length > 0 && results[0] == PackageManager.PERMISSION_GRANTED) {
-                        doLaunch(context, extraData);
-                    }
-
-                    return true;
+        List<String> permissionsList = new ArrayList<>();
+
+        PermissionListener listener = new PermissionListener() {
+            @Override
+            public boolean onRequestPermissionsResult(int i, String[] strings, int[] results) {
+                if (results.length > 0 && results[0] == PackageManager.PERMISSION_GRANTED) {
+                    JitsiMeetLogger.w(TAG + " POST_NOTIFICATION permission granted");
+                } else {
+                    JitsiMeetLogger.w(TAG + " POST_NOTIFICATION permission not granted");
                 }
-            };
+
+                return true;
+            }
+        };
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            permissionsList.add(RECORD_AUDIO);
+            permissionsList.add(POST_NOTIFICATIONS);
 
             JitsiMeetActivityDelegate.requestPermissions(
                 (Activity) context,
                 new String[]{POST_NOTIFICATIONS},
-                POST_NOTIFICATIONS_PERMISSION_REQUEST_CODE,
+                POST_NOTIFICATIONS_PERMISSIONS_REQUEST_CODE,
                 listener
             );
+
+            String[] permissionsArray = new String[ permissionsList.size() ];
+            permissionsArray = permissionsList.toArray( permissionsArray );
+
+
+            if (permissionsArray.length > 0) {
+                boolean permissionsResult
```

Comment:
@saghul I reworked the logic a bit.
I suggest we use the permission request listener only for POST_NOTIFICATIONS. That's it.
Both RECORD_AUDIO and POST_NOTIFICATIONS need to be granted.
We launch the service under the API >= 33 if all of the above check and we just launch for API < 33.